test/journal/test_dailyPass.ts

```
@@ -4,7 +4,22 @@ import {
+  normalizeTopicAction,
+  parseArchivistOutput,
+  computeJustCompletedSessions,
+  advanceJournalState,
+  parseJsonlEvents,
+  bucketParsedEvents,
+  type ParsedEntry,
 } from "../../server/journal/dailyPass.js";
+import type {
+  SessionExcerpt,
+  ExistingTopicSnapshot,
+  TopicUpdate,
```

ESLint

[error] Remove this unused import of 'TopicUpdate'.

(sonarjs/unused-import)